zone/lua_client.cpp

```diff
@@ -3308,6 +3308,38 @@ bool Lua_Client::RemoveAlternateCurrencyValue(uint32 currency_id, uint32 amount)
 	return self->RemoveAlternateCurrencyValue(currency_id, amount);
 }
 
+luabind::object Lua_Client::GetRaidOrGroupOrSelf(lua_State* L)
+{
+	auto t = luabind::newtable(L);
+	if (d_) {
+		auto self = reinterpret_cast<NativeType*>(d_);
+		auto l = self->GetRaidOrGroupOrSelf();
+		int i = 1;
+		for (const auto& e : l) {
+			t[i] = e;
```

Comment:
You'll probably need to be explicit on these and push `t[i] = Lua_Mob(e)`. Otherwise it'll just push the raw Mob pointer type and not the exported quest class. Might need to double check that previous buff patch for that too